### PR TITLE
[SSoC 2.0] Adding Readme for Frontend Projects

### PR DESCRIPTION
**Pull request by Avdhesh-Varshney**
Added Readme for Front end projects

Issue Number - 348

close issue #348

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
@shubhisingh184 @Vikash-8090-Yadav sir, please review this pr

**Comment by shubhisingh184** [comment]
@Avdhesh-Varshney there's a project music shop is removed, so update that.

**Comment by Avdhesh-Varshney** [comment]
@shubhisingh184 @Vikash-8090-Yadav please again review this pr

File: Frontend-Projects/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<div align="center">
+  <h1 align="center">
+    <img src="https://readme-typing-svg.herokuapp.com?color=45ffaa&size=40&width=900&height=80&lines=Welcome-to-the-World-of-Web3🚀"/>
```
Comment (Vikash-8090-Yadav) [inline Frontend-Projects/README.md:3]: it should be `Welcome-to-the-World-of-WebDev`

**Review by Vikash-8090-Yadav — CHANGES_REQUESTED** [review]
Do the necessary  changes.

**Comment by Avdhesh-Varshney** [comment]
> Do the necessary changes.

@Vikash-8090-Yadav @shubhisingh184 please again review this pr

**Pull request merged**